voicevox_engine/guided/extractor.py

```diff
@@ -0,0 +1,75 @@
+import os
+import snfa
+import numpy as np
+import pyworld as pw
+
+from typing import List, Optional
+from snfa.viterbi import Segment
+from scipy.signal import resample
+from .converter import conv2openjtalk
+from ..model import AudioQuery
+
+
+aligner: Optional[snfa.Aligner] = None
+model_path = "cv_jp.bin"
```

Comment:
It might be convenient if the value of `cv_jp.bin` could be specified as an argument.

Reply:
fixed.